client/packages.go

```diff
@@ -131,9 +132,10 @@ type FindOptions struct {
 
 	CommonID string
 
-	Section string
-	Private bool
-	Scope   string
+	Section  string
+	Category string
```

Comment:
these two probably should be swapped and Section have a comment that is deprecated